extensions/wikia/DesignSystem/styles/design-system.scss

```diff
@@ -0,0 +1,6 @@
+// These are temporary overrides that should be a part of DS resets
```

Comment:
After work we did in current sprint I think that we decided to keep those styles here.